pkg/policy/cidrgroup/cell.go

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cidrgroup
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/ipcache"
+	cilium_api_v2alpha1 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2alpha1"
+	"github.com/cilium/cilium/pkg/k8s/resource"
+)
+
+var Cell = cell.Module(
+	"fqdn-ingress-identities-manager",
+	"Identities manager for fqdn ingress policies",
+
+	cell.ProvidePrivate(func(ipcache *ipcache.IPCache) identitiesAllocator {
+		return ipcache
+	}),
+	cell.Invoke(registerManager),
+)
```

Comment:
To reiterate, we don't need this identity allocation code because the identity allocation should happen in the same place as it already does today when we process CIDR blocks in the CNP handler. We don't need to do additional identity allocation because the CIDRGroups are just syntactic sugar on top of CNPs.

Reply:
Ok, now I got it, thanks!